code/modules/projectiles/guns/energy.dm

```diff
@@ -89,13 +98,114 @@ GLOBAL_LIST_INIT(registered_cyborg_weapons, list())
 					return suit.cell
 	return null
 
+/obj/item/weapon/gun/energy/special_check(var/mob/user)
+
+	if(!..())
+		return
+	
+	if(hatch_open)
+		to_chat(user, "<span class='warning'>[src] has its cell cover still open!</span>")
+		return 0
+	return 1
+
+//Attempts to load A into src, depending on the type of thing being loaded and the load_method
```

Comment:
Inaccurate comment.

Reply:
Changed comment to be a bit more clear that it is for loading power cells